test/makeLatexReportRegression.m

```diff
@@ -0,0 +1,231 @@
+function makeLatexReportRegression(currentStatus, output, otherStatus)
+% generate a LaTeX report
+%
+% 
+    if ~exist('output','var')
```

Comment:
Actually, the function signature as is, would help merge this duplication of code into `makeLatexReport`, so left it out as is for now.